docs/HowTo/Quickstart.md

```diff
@@ -36,19 +36,19 @@ clienturl = "http://127.0.0.1:8888/"
 clientport = 8888
 publickeys = ["nodeKey.pub"]
 privatekeys = ["nodeKey.key"]
-passwords = "/<pathToPasswordFile>"/passwordFile
+passwords = "<PathToPasswordFile>"
```

Comment:
This has dropped the actual filename - ie, the updated version indicates this property is a path only which isn't correct

Reply:
Updated.